### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by robsonpeixoto** [jroesch/language-scala#14]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:565:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:187:43)
```

**Comment by laughedelic** [comment]
I think, https://github.com/jroesch/language-scala/pull/13 fixes it already

**Comment by mcous** [comment]
#12 has the fix as well

**Comment by robsonpeixoto** [comment]
Sorry the duplication

**Comment by gangstead** [comment]
Fixed in `v1.0.1`
